ex02.exs

```diff
@@ -81,24 +77,21 @@ end
 defmodule Ex02 do
 
   def new_counter(initial_value \\ 0) do
-    # « your code goes here»
+    {:ok, agent} = Agent.start_link(fn -> initial_value end)
+    agent
   end
 
   def next_value(agent) do
-    # « your code goes here»
+    Agent.get_and_update(agent, fn initial_value -> {initial_value, initial_value + 1} end )
   end
 
   @global_name :my_global_agent
 
   def new_global_counter(initial_value \\ 0) do
-    # « your code goes here»
-    # You can give an agent a name by adding the
-    # `name: @global_name` option to start_link
+    Agent.start_link(fn -> initial_value end, name: Global_agent)
```

Comment:
The idea was to use `name: @global_name` in both the start link and the get_and_update, so that the name is only defined in one place.

# -1